go-controller/pkg/libovsdbops/datapath.go

```diff
@@ -19,20 +18,18 @@ func findDatapathByPredicate(sbClient libovsdbclient.Client, lookupFcn func(dp *
 		return nil, err
 	}
 
-	if len(datapathBindings) != 1 {
-		return nil, fmt.Errorf("multiple datapath bindings found for a given lookup function")
-	}
-
-	return &datapathBindings[0], nil
+	return datapathBindings, nil
 }
 
-func FindDatapathByExternalIDs(sbClient libovsdbclient.Client, externalIDs map[string]string) (*sbdb.DatapathBinding, error) {
+func FindDatapathByExternalIDs(sbClient libovsdbclient.Client, externalIDs map[string]string) ([]sbdb.DatapathBinding, error) {
 	datapathLookupFcn := func(dp *sbdb.DatapathBinding) bool {
 		datapathMatch := false
 		for k, v := range externalIDs {
 			if itemVal, ok := dp.ExternalIDs[k]; ok {
 				if itemVal == v {
 					datapathMatch = true
+				} else {
```

Comment:
@astoycos could we just return `false` if one of the ExtIDs *doesn't* match? And then we could get rid of `datapathMatch` altogether and just return `true` at the end of the function because by that point all of them would have matched.

Reply:
Ack I'll go ahead and fix